dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -154,7 +154,7 @@ def docker_compose_tests(
     sys.exit(return_code)
 
 
-TEST_PROGRESS_REGEXP = r"tests/.*|providers/.*/tests/.*|task-sdk/tests/.*|.*=====.*"
+TEST_PROGRESS_REGEXP = r"tests/.*|providers/.*/tests/.*|task-sdk/tests/.*|airflow-ctl/tests/.*|.*=====.*"
```

Comment:
```suggestion
TEST_PROGRESS_REGEXP = r"tests/.*|providers/.*/tests/.*|task-sdk/tests/.*|airflow-ctl/tests/.*|.*=====.*"
```

Ach .. I noticed one missed change in my move - here and below. I will fix it separately :)

Reply:
My eyes still getting used to :) I see now what you will change :) I can also include it in this one for sure feel free to fix it separately :)

Reply:
Already merged #48100 :tada: